planner/core/plan_test.go

```diff
@@ -385,6 +385,33 @@ func (s *testPlanNormalize) TestExplainFormatHint(c *C) {
 		"use_index(@`sel_2` `test`.`t2` `idx_c2`), hash_agg(@`sel_2`), use_index(@`sel_1` `test`.`t1` `idx_c2`), hash_agg(@`sel_1`)"))
 }
 
+func (s *testPlanNormalize) TestHintRecoverable(c *C) {
```

Comment:
This only tests whether a plan containing TiFlash is recoverable. Would you like to update the name?

Reply:
OK.